### PR TITLE
[WIP] rely on spring to manage jwt encoding, deconding, refreshing ?

### PR DESCRIPTION
**Pull request by cmangeat**
making the things simplier, considering

https://stackoverflow.com/questions/39210683/how-inmemorytokenstore-works-with-spring-security-oauth2-and-is-this-the-safest/39662709#39662709

### CODE REVIEW

File: modules/services/authorizing/src/main/java/org/fao/geonet/authorizing/SecurityConfigurer.java [hunk 1]
```diff
@@ -9,31 +9,87 @@
...
+
+  @Bean
+  public TokenStore tokenStore() {
+    return new JwtTokenStore(accessTokenConverter());
+  }
+
+  @Bean
+  public JwtAccessTokenConverter accessTokenConverter() {
+    JwtAccessTokenConverter converter = new JwtAccessTokenConverter();
+    converter.setSigningKey("123");
```
Comment (cmangeat) [inline modules/services/authorizing/src/main/java/org/fao/geonet/authorizing/SecurityConfigurer.java:81]: here to set https://github.com/geonetwork/geonetwork-microservices/blob/main/modules/support-services/configuring/src/main/resources/config/geonetwork-prod.properties#L1 or better encrypting scheme

**Comment by fxprunayre** [comment]
FYI. I'm moving that PR to the new project structure and will resolve conflicts

**Comment by cmangeat** [comment]
note that this PR is in WIP state and that sole authorizing and searching modules have been "processed", at the end common security and other modules should be adapted also.

**Comment by fxprunayre** [comment]
Replaced by https://github.com/geonetwork/geonetwork-microservices/pull/20

**Pull request closed without merging**